circle.yml

```diff
@@ -0,0 +1,3 @@
+machine:
```

Comment:
If we are gonna use circleCI I think we should use V2 which requires the presence of the following file `.circleci/config.yml`

https://circleci.com/docs/2.0/language-ruby/